File: core-sample-app/src/main/java/com/pierfrancescosoffritti/aytplayersample/utils/PlaybackUtils.java
package com.pierfrancescosoffritti.aytplayersample.utils;

import java.util.Random;

public class PlaybackUtils {
    private static String[] videoIds = {"6JYIGclVQdw", "LvetJ9U_tVY", "S0Q4gqBUs7c", "zOa-rSM4nms"};
    private static Random random = new Random();

    public static String getNextVideoId() {
        return videoIds[random.nextInt(videoIds.length)];
    }
}

File: chromecast-sample-app/src/main/java/com/pierfrancescosoffritti/cyplayersample/utils/PlaybackUtils.kt
package com.pierfrancescosoffritti.cyplayersample.utils

import java.util.*

object PlaybackUtils {
    private val videoIds = arrayOf("6JYIGclVQdw", "LvetJ9U_tVY", "S0Q4gqBUs7c", "zOa-rSM4nms")

    fun getNextVideoId(): String {
        return videoIds[Random().nextInt(videoIds.size)]
    }
}